comit/src/ledger.rs

```diff
@@ -33,6 +33,33 @@ impl From<::bitcoin::Network> for Bitcoin {
     }
 }
 
+#[derive(Clone, Copy, Debug, PartialEq, Eq, Hash)]
+pub enum Lightning {
```

Comment:
Are we actually using this for anything now?

Reply:
rebase fail, I remove this in the first patch and put it back in in the second one - seems like I'm not reading my own commits before raising PRs :(